services/dynamic-sidecar/tests/unit/test_modules_mounted_fs.py

```diff
@@ -9,6 +9,8 @@
 import pytest
 from simcore_service_dynamic_sidecar.modules import mounted_fs
 
+pytestmark = pytest.mark.asyncio
```

Comment:
@sanderegg, there was a different way to set this now, right?

Reply:
well I think it depends in this case if the pytest-aiohttp library is installed. in which case I think you can just remove the mark.

Reply:
pytest-aiohttp is NOT installed but is pytest-asyncio. I thought now it was also automatic .. perhaps i misunderstood some comment you did a while ago